### ISSUE TITLE
Add keyboard support

### ISSUE THREAD

**Issue by fer** [silvestrevivo/javascript-calculator#1]
It'd be lovely to have some keyboard support, so we can use normal keypress as another input method.

**Comment by silvestrevivo** [comment]
That is true, your are right. I would like to add some improvements in the next weeks, as well as new branches to make different solutions in JS!. Thanks for your comment Fernando!! (you can as well your personal website improve heheheheh

**Comment by fer** [comment]
sure! feel free to open a ticket or propose a PR ;)

**Comment by silvestrevivo** [comment]
Do you have a repo in Github about the site? And a design or idea? Let's do it!
